### PR TITLE
ExcerciseTicket | master

### PR DESCRIPTION
**Pull request by PablitoBonito**
linkToTheTicket

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/AddBlogInPageLevel.testcase [hunk 1]
```diff
@@ -0,0 +1,42 @@
+definition {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/AddBlogInPageLevel.testcase:1]: The commit message should add context to what the commit is doing.

ie: `LRQA-61956 - Add test to add blog via portlet`

File: portal-web/test/functional/com/liferay/portalweb/tests/AddBlogInPageLevel.testcase [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "User Interface";
+	property testray.main.component.name = "User Interface";
+	
+	
+	setUp {
+	User.firstLoginPG();
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/AddBlogInPageLevel.testcase:10]: There needs to be another tab indent here and throughout the test. Please take a look at other testcases for examples of formatting like we went over before in the 1st exercise. Running ant format source would usually catch these.

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by PablitoBonito** [comment]
ci:test:relevant

**Comment by PablitoBonito** [comment]
ci:test:sf

**Comment by john-co** [comment]
please see additional SF in https://github.com/john-co/liferay-portal/pull/350.

There seems to have been something wrong with the source-formatter on your local machine. I was able to fix the issue we saw earlier by rebasing this branch to with master.

**Pull request closed without merging**